places-solution-integrations/README.md

```diff
@@ -0,0 +1,23 @@
+# Places Solution Integrations
+
+By itself, using Adobe Places to collect location data isn't very useful in your app. You will start to get real value from location data when you provide it to your other Adobe solutions. This will allow you to use the location data to deliver custom experiences and make informed decisions.
```

Comment:
- Please reword as follows:
"Using Places alone to collect location data is not useful in your app. The real value comes from the location data that you provide to other Adobe solutions. You can use this data to deliver custom experiences and make informed decisions."